packages/components/src/MultiSelect/subcomponents/Popover/Popover.module.scss

```diff
@@ -11,7 +11,7 @@
   background: $color-white;
   overflow: auto;
 
-  &:focus {
+  &:focus-visible {
     outline: none;
```

Comment:
Should be like this instead right?

```suggestion
  &:focus {
    outline: none;
  }
    
  &:focus-visible {
```

Reply:
It didn't seem to be adding an outline on Chrome or Firefox but I've committed your suggestion anyway just in case.

Reply:
My guess is that browsers don't do an outline on a div:focus so this line wasn't actually needed anyway, but not bad to have just in case.